tests/test_header_matching.js

```diff
@@ -487,29 +487,34 @@ test('done does not fail when specified request header is not missing', t => {
   )
 })
 
-test('done fails when specified bad request header is present', t => {
-  nock('http://example.test', {
+test('when badheaders are present, badheaders prevents match', async t => {
+  const scope = nock('http://example.test', {
     badheaders: ['cookie'],
   })
-    .post('/resource')
+    .post('/')
     .reply(200, { status: 'ok' })
```

Comment:
Maybe update this too to an empty call, `.reply()`, since the args don't provide anything to the test?

Reply:
Done, also changed to `.get()`.